builtin/mainmenu/dlg_create_world.lua

```diff
@@ -78,6 +78,17 @@ local mgv6_biomes = {
 	},
 }
 
+local mapgens_descriptions = {
+	v7 = fgettext("Default mapgen with large complex mountins and plains."),
+	valleys = fgettext("Large valleys with complex terrain and rivers."),
+	carpathian = fgettext("Realistic looking world with vast plains."),
+	v5 = fgettext("Old mapgen."),
+	flat = fgettext("World Flat terrain."),
```

Comment:
maybe you meant this?
```suggestion
	flat = fgettext("Flat world terrain"),
```

Reply:
yeah...oops